client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -521,3 +554,206 @@ const TabReportsContent: React.FC<{
     </>
   );
 };
+
+const TabTasksContent: React.FC<{
```

Comment:
Ideally we should be able to reuse the code from `TasksPage`. In the current state this requires a refactoring that would extract the "base" component. The best strategy would be to use a different PR for that and merge it first. This way we (and our test suite) ensure that the refactoring caused no regression.

After the refactoring both `TasksPage` and `TabTasksContent` should use the base and provide the custom parts. There are few differences:
1. identifiers i.e. `tableName`, `persistenceKeyPrefix`
2. visible columns - note that hiding (via `initialColumns`)  looks easier then maintaining 2 lists of columns
3. query params - `implicitFilters`
4.  possibly more

If "base" component variant would be not achievable we can break the TasksPage into smaller parts and try to reuse those.

Reply:
@rszwajko Thank you for the feedback! I would really like to continue working on this task, but I currently have a challenge. My internship with the company has concluded, and I no longer have access to the computer where the product environment was properly set up. Unfortunately, running Konveyor on Minikube in the Windows operating system presents a significant issue: the local Konveyor UI is inaccessible.
 If the issue with installing Konveyor is resolved, I’d be happy to continue working on this task. Please let me know.